src/main.cc

```diff
@@ -766,6 +773,16 @@ main(int argc, char **argv)
 	if (strlen(tarantool_path) < strlen(tarantool_bin))
 		panic("executable path is trimmed");
 
+	if (script == NULL && !(opt_mask & O_INTERACTIVE)) {
```

Comment:
Should we test new behavior?

Reply:
I believe this is tested via [this test](https://github.com/tarantool/tarantool/blob/fb040252a51d590a3e9c3a0afe6a2560b069aaec/test/app-tap/gh-5040-inter-mode-isatty.test.lua#L35-L38) case, that is updated respectively in the patch, isn't it? All conditions are met: no args (hence no script provided) and interactive mode is disabled. Otherwise do you suggest to add a separate test to check that Tarantool yields usage message for this case?

Reply:
> I believe this is tested via [this test](https://github.com/tarantool/tarantool/blob/fb040252a51d590a3e9c3a0afe6a2560b069aaec/test/app-tap/gh-5040-inter-mode-isatty.test.lua#L35-L38) case, that is updated respectively in the patch, isn't it? All conditions are met: no args (hence no script provided) and interactive mode is disabled.

You are right. But I don't like that testing of default behavior is a part of regression test for #5040. It is not obvious that we test Tarantool's default behavior in an updated testcase. Moreover, a patch for an existed test doesn't check the error message.

> Otherwise do you suggest to add a separate test to check that Tarantool yields usage message for this case?

I would add a separate test, but I will not insist.

Reply:
Added the separate file with the tests for the new options.

Reply:
Looks resolved. Resolving the thread so.